src/followers/followers.js

```diff
@@ -0,0 +1,112 @@
+import { useState, useEffect } from 'react';
+import apiFetch from '@wordpress/api-fetch';
+import { addQueryArgs } from '@wordpress/url';
+import { __ } from '@wordpress/i18n';
+import { Pagination } from './pagination';
+import { ExternalLink } from '@wordpress/components';
+
+const { namespace } = window._activityPubOptions;
+
+function getPath( userId, per_page, order, page ) {
+	const path = `/${ namespace }/users/${ userId }/followers`;
+	const args = {
+		per_page,
+		order,
+		page,
+		context: 'full'
+	};
+	return addQueryArgs( path, args );
+}
+
+function usePage() {
+	const [ page, setPage ] = useState( 1 );
+	return [ page, setPage ];
+}
+
+export function Followers( {
+	selectedUser,
+	per_page,
+	order,
+	title,
+	page: passedPage,
+	setPage: passedSetPage,
+	className = ''
+} ) {
+	const userId = selectedUser === 'site' ? 0 : selectedUser;
+	const [ followers, setFollowers ] = useState( [] );
+	const [ pages, setPages ] = useState( 0 );
+	const [ total, setTotal ] = useState( 0 );
+	const [ localPage, setLocalPage ] = usePage();
+	const page = passedPage || localPage;
+	const setPage = passedSetPage || setLocalPage;
+	const prevLabel = (
+		<>
+			<span class="wp-block-query-pagination-previous-arrow is-arrow-arrow" aria-hidden="true">
+				{
+					/* translators: arrow for previous followers link */
+					__( '←', 'activitypub' )
+				}
+			</span>
+			{ __( 'Less', 'activitypub' ) }
```

Comment:
Maybe this (and the below) could be combined into a single translatable string, with `createInterpolateElement` from `@wordpress/element`. It would make things easier in RTL languages, for example.

Something like this (not tested):

```diff
diff --git a/src/followers/followers.js b/src/followers/followers.js
index aee1c3c..b756f8f 100644
--- a/src/followers/followers.js
+++ b/src/followers/followers.js
@@ -1,6 +1,7 @@
 import { useState, useEffect } from 'react';
 import apiFetch from '@wordpress/api-fetch';
 import { addQueryArgs } from '@wordpress/url';
+import { createInterpolateElement } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 import { Pagination } from './pagination';
 import { ExternalLink } from '@wordpress/components';
@@ -39,27 +40,19 @@ export function Followers( {
 	const [ localPage, setLocalPage ] = usePage();
 	const page = passedPage || localPage;
 	const setPage = passedSetPage || setLocalPage;
-	const prevLabel = (
-		<>
-			<span class="wp-block-query-pagination-previous-arrow is-arrow-arrow" aria-hidden="true">
-				{
-					/* translators: arrow for previous followers link */
-					__( '←', 'activitypub' )
-				}
-			</span>
-			{ __( 'Less', 'activitypub' ) }
-		</>
+	const prevLabel = createInterpolateElement(
+		/* translators: arrow for previous followers link */
+		__( '<span>←</span> Less', 'activitypub' ),
+		{
+			span: <span class="wp-block-query-pagination-previous-arrow is-arrow-arrow" aria-hidden="true" />,
+		}
 	);
-	const nextLabel = (
-		<>
-			{ __( 'More', 'activitypub' ) }
-			<span class="wp-block-query-pagination-next-arrow is-arrow-arrow" aria-hidden="true">
-				{
-					/* translators: arrow for next followers link */
-					__( '→', 'activitypub' )
-				}
-			</span>
-		</>
+	const nextLabel = createInterpolateElement(
+		/* translators: arrow for next followers link */
+		__( 'More <span>→</span>', 'activitypub' ),
+		{
+			span: <span class="wp-block-query-pagination-next-arrow is-arrow-arrow" aria-hidden="true" />,
+		}
 	);
 
 	useEffect( () => {
@@ -109,4 +102,4 @@ function Follower( { name, icon, url, preferredUsername } ) {
 			</span>
 		</ExternalLink>
 	)
-}
\ No newline at end of file
+}

```

Reply:
TIL about `createInterpolateElement` - thanks so much for this, great point about RTL and just keeping the context together.